### ISSUE TITLE
Line blocks and the text writer

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1109]
The patch for issue #1028 fixes the initial issue of line blocks and the text writer. However, it is incomplete.

With:

```
Test
====

* one

  | need
  | specific
  | linebreaks

* two

  | more
  | linebreaks

No longer in the list.
```

The HTML output is as intended. The latexpdf has two blank lines worth of whitespace before and after each line block, which is less than ideal, but could be considered acceptable. However, the text writer emits this:

```
Test
****

* one

     need
     specific
     linebreaks
* two

     more
     linebreaks
No longer in the list.
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1109
- Originally reported by: Ewan Edwards
- Originally created at: 2013-02-14T17:48:42.475

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-03-09 15:04:58+00:00_

Add regression test for #1109. Issue was corrected at previous commit 62b0a5f17f62. Closes #1109

→ <<cset e2087189ead6c44093be7e6c3de1dfb52624867f>>

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-03-09 15:07:37+00:00_

Thanks!

(BTW 4ff0575 is correct instead of 62b0a5f17f62.)
